test/functional/data/invalid_txs.py

```diff
@@ -122,7 +123,8 @@ class SizeTooSmall(BadTxTemplate):
     def get_tx(self):
         tx = CTransaction()
         tx.vin.append(self.valid_txin)
-        tx.vout.append(CTxOut(0, CScript([OP_TRUE])))
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN] * (MIN_PADDING - 1))))
```

Comment:
Wouldn't it be better to create an output that passes standardness checks here so that it's only failing for one reason?

Reply:
Sorry what's non-standard about this output? It's an OP_RETURN(so it can be dust), all pushes, under 80 bytes...

Reply:
OP_RETURN is probably not a push?

```
decodescript 6a6a6a6a6a6a

{
  "asm": "OP_RETURN OP_RETURN OP_RETURN OP_RETURN OP_RETURN OP_RETURN",
  "desc": "raw(6a6a6a6a6a6a)#6j3qx6z8",
  "type": "nonstandard"
}

Reply:
Uhm, OP_RETURN isn't a push, gibbs... *facepalm*

Reply:
could add OP_RESERVED to do a bit of trolling